### PR TITLE
PWA theme colour match dark/light themes

### PR DESCRIPTION
**Pull request by jtsorlinis**
# Changelog Entry

### Description

- When using open-webui as a PWA the bottom and top bars are always white, which doesn't look great if using dark modes.

### Added

- PWA theme colour should now be light or dark depending on the selected theme.

### CODE REVIEW

File: src/app.html [hunk 1]
```diff
@@ -38,15 +40,21 @@
...
 					document.documentElement.classList.add('her');
 				} else {
 					document.documentElement.classList.add('dark');
 				}
 
+				if (
+					localStorage?.theme.includes('dark') ||
+					(localStorage.theme === 'system' && isSystemDark)
+				) {
+					window.themeColor.content = '#0d0d0d';
```
Comment (tjbck) [inline src/app.html:55]: Shouldn't this also be added to the listener?

**Comment by erickgodev** [comment]
Look, I was going to implement this but I had problems with the OLED and the HUR making the iPhone not recognize the current theme, oh you have to edit it in the settings too because if you change the theme it will only appear when you reload the app

**Comment by parthpat12** [comment]
The issue still persists on iOS 17.5.1. I’m on latest v0.3.15 and the white status bars at top still show. Can we reopen this issue since it’s not fixed? The text on the bottom is also getting covered by swipe bar.
Screenshot 2024-08-26 at 3 18 50 PM

**Comment by jtsorlinis** [comment]
Yeah I'm happy to make the suggested changes but someone closed the PR already

**Comment by tjbck** [comment]
Just reopened, but I'd encourage you to rebase first!

File: src/app.html (hunk 1 above)
Comment (jtsorlinis) [reply src/app.html:55]: I've added it to the listener!

**Comment by jtsorlinis** [comment]
I've added it to the listener, this won't work perfectly for black vs dark grey but having it as dark grey still looks a lot better than bright white

**Comment by tjbck** [comment]
Is this PR tested? Could you provide us with a video demo?

**Pull request closed without merging**